docs/Tasks/triggers/private-base-image-update.md

```diff
@@ -0,0 +1,84 @@
+# Track base image update from any Azure Container Registry
+
+ACR Tasks supports automated builds for when a container's base iamge is updated. Previously, these automated builds were supported for base images in public repositories, such as DockerHub, or from base images from a task's home registry. Now ACR Tasks can also track base image updates from another ACR than the task's. This base image registry can be **any Azure Container Registry, anywhere in the world**, even geo-replicated. This tutorial combines much of the steps seen in the existing tutorials for [base image update](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-tutorial-base-image-update) and [cross-registry auth](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-tasks-cross-registry-authentication).
+
+
+# Prerequisites
+
+In this tutorial you will need to first create two registries in different regions.
+* The first registry (`homeRegistry`) will be used to create and execute a task. Create this registry in West US.
+* The second registry (`baseRegistry`) will host a base image that will be used by the task to build an image. Create this registry in East US.
+
+
+## Prepare Base Registry
+
+Fork the repo https://github.com/Azure-Samples/acr-build-helloworld-node.git
+
+In the current directory build and push the base image to the base registry
+
+```
+az acr build --image baseimages/node:9-alpine --registry baseRegistry --file Dockerfile-base .
+```
+
+## Create a task that tracks the private base image
+
+Create a task in the home registry with a system-assigned identity. This task depends on the base image built above.
+
+```
+az acr task create \
+  --registry homeRegistry \
+  --name hometask \
+  --image helloworld:v2 \
+  --context https://github.com/$GIT_USER/acr-build-helloworld-node.git \
+  --file Dockerfile-app \
+  --git-access-token $GIT_PAT \
+  --arg REGISTRY_NAME=baseRegistry.azurecr.io \
```

Comment:
ACR_NAME=${BASE_REGISTRY} \

Reply:
`REGISTRY_NAME` is an argument in Dockerfile-app. I will address the other comments about env vars and formatting.